lib/routes/note_page.dart

```diff
@@ -18,13 +18,15 @@ import 'package:potato_notes/internal/app_info.dart';
 import 'package:potato_notes/internal/note_colors.dart';
 import 'package:potato_notes/internal/preferences.dart';
 import 'package:potato_notes/internal/utils.dart';
+import 'package:potato_notes/locator.dart';
 import 'package:potato_notes/routes/note_page_image_gallery.dart';
 import 'package:potato_notes/widget/note_color_selector.dart';
 import 'package:potato_notes/widget/note_toolbar.dart';
 import 'package:potato_notes/widget/note_view_images.dart';
 import 'package:provider/provider.dart';
 import 'package:rich_text_editor/rich_text_editor.dart';
 import 'package:spicy_components/spicy_components.dart';
+import 'package:uuid/uuid.dart';
```

Comment:
unused import

Reply:
Yeah this made my build error actually. forgot to commit it